From a909015801d948871355365246115b03abb4c56a Mon Sep 17 00:00:00 2001
From: Megha Dey <megha.dey@linux.intel.com>
Date: Mon, 6 Jun 2016 16:01:14 -0700
Subject: [PATCH] ndctl: fix build on systems where /bin/sh is not a link to
 bash

On Debian and other Debian-based Linux distributions, autogen.sh
gives the following error:
    ./git-version: 37: ./git-version: Bad substitution

This happens because the git-version script is using a 'bashim' to
perform a substring extraction, which can only be performed
using bash. Hence shifting from sh to bash.

Signed-off-by: Megha Dey <megha.dey@linux.intel.com>
---
 git-version | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/git-version b/git-version
index c78aff4..629f6d4 100755
--- a/git-version
+++ b/git-version
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 dirty() {
 	git update-index -q --refresh
-- 
1.9.1

